tests/conftest.py

```diff
@@ -354,6 +354,43 @@ def finalizer():
     return factory
 
 
+@pytest.fixture()
+def teardown_factory(request):
+    """
+    Tearing down a resource that was created during the test
+    To use this factory, you'll need to pass 'teardown_factory' to you test
```

Comment:
thanks for the docstring, one minor - s/you/your and one more instance below.  When you push new changes, I will use previous approval and merge it.

Reply:
done